patches/minecraft/net/minecraft/tileentity/TileEntityBrewingStand.java.patch

```diff
@@ -76,3 +76,28 @@
      }
  
      public boolean[] func_174902_m()
+@@ -386,4 +393,24 @@
+             this.field_145945_j[i] = null;
+         }
+     }
++
++    net.minecraftforge.items.IItemHandler handlerInput = new net.minecraftforge.items.wrapper.SidedInvWrapper(this, net.minecraft.util.EnumFacing.UP);
++    net.minecraftforge.items.IItemHandler handlerOutput = new net.minecraftforge.items.wrapper.SidedInvWrapper(this, net.minecraft.util.EnumFacing.DOWN);
++
++    @Override
++    public <T> T getCapability(net.minecraftforge.common.capabilities.Capability<T> capability, net.minecraft.util.EnumFacing facing)
++    {
++        if (facing != null && capability == net.minecraftforge.items.CapabilityItemHandler.ITEM_HANDLER_CAPABILITY)
++        {
++            if (facing == net.minecraft.util.EnumFacing.UP)
```

Comment:
This is a patch file remember you shouldnt be that verbose with your braces ;)
if (facing == net.minecraft.util.EnumFacing.UP) return (T)handlerInput;